docker_explorer/explorer.py

```diff
@@ -204,14 +204,14 @@ def GetContainersJson(self, only_running=False):
 
       mount_points = container_object.GetMountpoints()
       if mount_points:
+        container_json['mount_points'] = []
```

Comment:
if you add the setdefault you can remove this and the `if mount_points:` before, the loop would be enough in that case, `GetMountpoints` returns an empty list in no entries and if it is an empty list the for loop will pass

Reply:
yes, fixed